model/comments-model.js

```diff
@@ -0,0 +1,16 @@
+const db = require("../db/connection");
+
+function fetchComments(article_id, order = "desc", sort_by = "created_at") {
```

Comment:
default values can't ever be changed here so might as well just put in the sql query instead of trying to be fancy haha